### PR TITLE
Add VBA grammar

### PR DESCRIPTION
**Pull request by DecimalTurn**
This PR replaces the current VBA grammar that is actually for VB.NET by a grammar specific to VBA.

See also https://github.com/github-linguist/linguist/pull/6436 and https://github.com/serkonda7/vscode-vba/issues/76

## Checklist:

- [x] **I am changing the source of a syntax highlighting grammar**
  - Old: https://github.com/peters-ben-0007/VBDotNetSyntax
  - New: https://github.com/tiabeast/vscode-vba-json

### CODE REVIEW

**Comment by DecimalTurn** [comment]
This PR is now ready for review. 

Note that the submodule update was needed to fix [these errors](https://github.com/serkonda7/vscode-vba/issues/76#issuecomment-1575351421).

Also thanks to @serkonda7 for the help on this and for supplying the grammar.

**Review by serkonda7 — APPROVED** [review]
Looks good to me.

File: grammars.yml [hunk 1]
```diff
@@ -1144,6 +1144,8 @@ vendor/grammars/vscode-singularity:
 - source.singularity
 vendor/grammars/vscode-slice:
 - source.slice
+vendor/grammars/vscode-vba-json:
+- source.vba
```
Comment (lildude) [inline grammars.yml:1148]: Looks like you might have modified this file directly. There is no need as it is maintained by various scripts. In this case, this repo has two grammar files and the `add-grammar` script will have added two lines to this file:

```suggestion
- source.vba
- source.wwb
```

**Review by lildude — APPROVED** [review]

File: grammars.yml (hunk 1 above)
Comment (DecimalTurn) [reply grammars.yml:1148]: Thanks! Weirdly, I did run the `add-grammar` script on a separate branch for comparison before submitting the PR for review, but it didn't add the lines to `grammar.yml`. However, I cleaned the repo this morning, tried it again and the lines were added this time 🤷‍♂️.

**Pull request merged**